http/vertx/src/test/java/io/quarkus/ts/http/minimum/AbstractIT.java

```diff
@@ -0,0 +1,121 @@
+package io.quarkus.ts.http.minimum;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.testcontainers.shaded.org.hamcrest.MatcherAssert.assertThat;
+import static org.testcontainers.shaded.org.hamcrest.Matchers.containsString;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+
+public abstract class AbstractIT {
+
+    @Test
+    public void httpServerAndMetrics() {
+        requests().get("/hello").then()
+                .statusCode(HttpStatus.SC_OK)
+                .body("content", is("Hello, World!"));
+
+        Response response = requests().get("/q/metrics");
+        assertEquals(HttpStatus.SC_OK, response.statusCode());
+        assertThat(response.getContentType(), containsString("application/openmetrics-text"));
+        String body = response.body().asString();
+        Map<String, Metric> metrics = parseMetrics(body);
+        assertTrue(metrics.containsKey("worker_pool_active"));
+        assertTrue(metrics.containsKey("worker_pool_completed_total"));
+        assertTrue(metrics.containsKey("worker_pool_queue_size"));
```

Comment:
Maybe we could also check the following metrics:
`worker_pool_queue_delay_seconds_max`
`worker_pool_queue_delay_seconds_count`
`worker_pool_queue_delay_seconds_sum`
`worker_pool_usage_seconds_count`
`worker_pool_usage_seconds_sum`
`worker_pool_ratio`
`worker_pool_idle`

Reply:
They all go together anyway, and we cannot even reliably check the content of these. I am not sure additional checks will play a role there.